blocks/library/embed/index.js

```diff
@@ -114,11 +123,15 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 							return;
 						}
 						response.json().then( ( obj ) => {
-							const { html, type } = obj;
+							const { html, type, provider_name: providerName } = obj;
+							const providerNameSlug = kebabCase( toLower( providerName ) );
```

Comment:
Does `kebabCase` not handle `toLower` already?

Reply:
No, afaik `kebabCase` handles `lowerCase`, not `toLower`, which leads to ugly transformations from `YouTube` to `you-tube`.